src/style.css

```
@@ -1,3 +1,107 @@
-body {
+* {
   margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+li {
+  margin-left: 20px;
+}
+
+/* Use Roboto font */
+body {
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+}
+
+/* Header styling */
+header {
+  background-color: #fff;
+  justify-content: space-between;
+  width: 100%;
+  display: flex;
+  align-items: center;
+  padding: 0 50px;
+}
+
+/* Logo styling */
+.logo img {
+  width: 40px;
+  height: 40px;
+  display: block;
+}
+
+/* Navigation styling */
+.nav-list {
+  display: flex;
+  list-style: none;
+}
+
+.nav-item {
+  display: flex;
+  justify-content: center; /* Центрування по горизонталі */
```

```suggestion
  justify-content: center;
```
It's clear from the code
No need to add a lot of comments - they are required in complicated or unclear cases. I'd say in this task everything is clear from the code - no need to overload it with comments